### PR TITLE
Fix part of #2833: Added named arguments in CustomBulletSpanTest

### PR DESCRIPTION
**Pull request by arinarora18**
## Explanation

Fix part of #2833: Added missing named arguments in CustomBulletSpanTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt [hunk 1]
```diff
@@ -80,7 +80,7 @@ class CustomBulletSpanTest {
   }
   private val testStringWithCustomBulletSpan = SpannableString("Text With \nBullet Point").apply {
     this.setSpan(
-      CustomBulletSpan(context),
+      CustomBulletSpan(context = context),
       10,
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt:84]: please add the named argument here too. 

We usually add named argument where we are passing the arguments and just by reading the argument we cannot understand what is the use of that particular argument. So, here we are passing 10 and 20, which is not helpful to understand what it is, so we need named argument here.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Comment by anandwana001** [comment]
@arinarora18 Please follow the first and second point from the checklist.

```
The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
 The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
```

File: app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt (hunk 1 above)
Comment (arinarora18) [reply app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt:84]: but "setSpan" is not a kotlin function,  so named arguments are not allowed in it.

**Review by arinarora18 — COMMENTED** [review]
"setSpan" is a java funtion and I think named arguments can't be passed in it.

File: app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt:84]: ah, this is catch here. Thanks for this update.

**Comment by anandwana001** [comment]
We missed this one. Thanks for the update. I am closing this issue and removing this file from the list. Let me know if you want to work on another file from the given list in the issue.

**Pull request closed without merging**